modules/nw-dns-forward.adoc

```diff
@@ -55,6 +63,9 @@ spec:
 ----
 <1> Must comply with the `rfc6335` service name syntax.
 <2> Must conform to the definition of a subdomain in the `rfc1123` service name syntax. The cluster domain, `cluster.local`, is an invalid subdomain for the `zones` field.
+ifdef::openshift-rosa,openshift-dedicated[]
+Only forward to specific zones, such as your intranet. You must specify at least one zone. Otherwise, your cluster can lose functionality.
```

Comment:
I didn't see this part in the linked preview doc
https://59695--docspreview.netlify.app/openshift-rosa/latest/networking/dns-operator.html